frontend/src/components/Worksheet/URLExportButton.tsx

```diff
@@ -6,18 +6,26 @@ import type { ExoticWorksheet } from '../../slices/WorksheetSlice';
 import { useStore } from '../../store';
 
 export default function URLExportButton() {
-  const { viewedSeason, viewedWorksheetName, courses } = useStore(
+  const { viewedSeason, viewedWorksheetName, courses, user } = useStore(
     useShallow((state) => ({
       viewedSeason: state.viewedSeason,
       viewedWorksheetName: state.worksheetMemo.getViewedWorksheetName(state),
       courses: state.courses,
+      user: state.user,
     })),
   );
 
   async function handleExport() {
+    // Format creator name: "FirstName LastName" or fallback to netId
+    const creatorName =
+      user?.firstName && user.lastName
+        ? `${user.firstName} ${user.lastName}`
+        : (user?.netId ?? 'Unknown');
```

Comment:
Can this happen...? I'd rather not imagine `user` being null _and_ they can interact with this function.